mavisp/modules.py

```diff
@@ -299,6 +299,42 @@ def ingest(self, mutations):
             raise MAVISpMultipleError(warning=warnings,
                                       critical=[])
 
+class SAS(DataType):
+
+    module_dir = "sas"
+    name = "sas"
+
+    def ingest(self, mutations):
+        warnings = []
+
+        sas_file = os.listdir(os.path.join(self.data_dir, self.module_dir))
+        if len(sas_file) != 1:
+            this_error = f"multiple or no files found in {sas_file}; only one expected"
+            raise MAVISpMultipleError(warning=warnings,
+                                      critical=[MAVISpCriticalError(this_error)])
+
+        sas_file = sas_file[0]
+
+        log.info(f"parsing sas file {sas_file}")
+
+        try:
+            rsa = pd.read_fwf(os.path.join(self.data_dir, self.module_dir, 'sasa.rsa'),
+                skiprows=4, skipfooter=4, header=None, widths=[4,4,1,4,9,6,7,6,7,6,7,6,7,6],
+                names = ['entry', 'rest', 'chain', 'resn', 'all_abs', 'sas_all_rel', 'sas_sc_abs',
+                'sas_sc_rel', 'sas_mc_abs', 'sas_mc_rel', 'sas_np_abs', 'sas_np_rel', 'sas_ap_abs',
+                'sas_ap_rel'],
+                usecols = ['rest', 'resn', 'sas_sc_rel'],
+                index_col = 'resn').fillna(pd.NA)
+            self.data = rsa.rename(columns={'sas_sc_rel': 'Relative Side Chain Accessibility'})
```

Comment:
when I run this on a test case, I get all `nan`s in my accessibility column. I think this is because we should also have a column called `mutation` which contains the mutation in the `[WT][number][mut]` format (.e.g. `A34C`), otherwise we can merge it with the final data frame correctly. Notice that this mutation column has the same mutations as the `mutations` argument of this function. At the end you need to have a datafrmae like:

```
mutation | Relative side...
A3N        | 50.0
A3C.       | 50.0
A9F         | 60.0
...
```